src/unittests/f32_interval.spec.ts

```diff
@@ -681,16 +681,20 @@ g.test('acosInterval')
     // prettier-ignore
     [
       // Some of these are hard coded, since the error intervals are difficult to express in a closed human readable
-      // form due to the inherited nature of the errors.
+      // form due to the irrational nature of the values.
```

Comment:
nit: "non-sensical" instead of irrational?

Reply:
Changed the text to 'due to the complexity of their derivation', as I did in the asin PR